src/main/java/com/app/library/Application.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableAsync;
 
 @SpringBootApplication
+@EnableAsync
```

Comment:
until you don't use `@Async`, that annotation is not needed :)

Reply:
Annotation Removed